src/backend/distributed/commands/utility_hook.c

```diff
@@ -431,24 +431,25 @@ ProcessUtilityInternal(PlannedStmt *pstmt,
 	{
 		AlterSubscriptionStmt *alterSubStmt = (AlterSubscriptionStmt *) parsetree;
 		if (!superuser() &&
-			StringStartsWith(alterSubStmt->subname,
-							 SHARD_MOVE_SUBSCRIPTION_PREFIX))
+			StringStartsWith(alterSubStmt->subname, "citus_"))
```

Comment:
This is an important functional change for the security of shard splits.

Reply:
It would be helpful if you can explain why are prefixing it with "citus_" for my understanding. If it is already mentioned and I missed it would be helpful to point it out.

Reply:
added a comment about it